ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/P2TargetUtils.java

```diff
@@ -1416,33 +1416,24 @@ private Collection<URI> getArtifactRepositories(ITargetDefinition target) throws
 				result.addAll(getArtifactRepositories(referencedTargetDefinition));
 			}
 		}
-		if (useAdditionalLocalArtifacts()) {
-			// get all the artifact repos we know in the manager currently
-			result.addAll(Arrays.asList(manager.getKnownRepositories(IRepositoryManager.REPOSITORIES_ALL)));
 
-			if (Boolean.parseBoolean(System.getProperty("pde.usePoolsInfo", "true"))) { //$NON-NLS-1$ //$NON-NLS-2$
-				try {
-					result.addAll(RepositoryHelper.getSharedBundlePools().stream().map(Path::toUri).toList());
-				} catch (Exception e) {
-					//$FALL-THROUGH$
-				}
-			}
+		// get all the local artifact repos we know in the manager currently
+		result.addAll(Arrays.asList(manager.getKnownRepositories(IRepositoryManager.REPOSITORIES_LOCAL)));
 
-			// Add in the IDE profile bufindProfileReposndle pool and all known
-			// workspaces
-			findProfileRepos(result);
-			findWorkspaceRepos(result);
+		if (Boolean.parseBoolean(System.getProperty("pde.usePoolsInfo", "true"))) { //$NON-NLS-1$ //$NON-NLS-2$
+			try {
+				result.addAll(RepositoryHelper.getSharedBundlePools().stream().map(Path::toUri).toList());
+			} catch (Exception e) {
+				//$FALL-THROUGH$
+			}
 		}
-		return result;
-	}
 
-	/**
-	 * return whether or not to use local artifact repositories when provisioning the target
-	 */
-	private boolean useAdditionalLocalArtifacts() {
-		// XXX consider using a preference here or another strategy if users are able to spec
-		// what local repos are to be considered.
-		return true;
+		// Add in the IDE profile bufindProfileReposndle pool and all known
```

Comment:
`bufindProfileReposndle` ?

Reply:
That crap was there before. 😱 

https://github.com/eclipse-pde/eclipse.pde/blob/e7d05ff7dc03989b3da04b075feb3962444e0612/ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/P2TargetUtils.java#L1431

Reply:
> That crap was there before. 😱

You added it in your previous commit 🙈
Before that it was: `// Add in the IDE profile bundle pool and all known workspaces`